app/Controllers/feedController.php

```diff
@@ -564,21 +575,21 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 			}
 
 			if ($simplePie != null) {
-				if ($feed->name(true) == '') {
+				if ($feed->name(true) === '') {
```

Comment:
It does not matter much here, but this is another example where `===` is less safe.
Imagine we change `->name()` to return `null` if undefined or `false` in case of error (which is quite common), and the new code will lead to a bug, while the previous code was safe.

Reply:
It doesn't matter much here, but this is another example where === is less safe.
Let's say we change ->name() so that it returns null if undefined or false on error (which is quite common), and the new code will lead to a bug, whereas the previous code was safe.

Here is the function that is called when we do ->name();

```php
public function name(bool $raw = false): string {
		return $raw || $this->name != '' ? $this->name : preg_replace('%^https?://(www[.])?%i', '', $this->url);
	}
```

In any case we have a string in output if the function, what you do not want is that your string is of type: ""

Reply:
I know what we have now. My point was that it could change, and null / false are quite common